jest.config.js

```diff
@@ -326,7 +326,10 @@ const jestConfig = {
         // Not this file itself
         '!jest.config.js',
         // Exclude deprecated components from coverage report
-        '!**/venia-ui/lib/components/Checkout/**'
+        '!**/venia-ui/lib/components/Checkout/**',
+        // Not GraphQL files, gql.ce.js or gql.ee.js
+        '!**/*.gql.js',
+        '!**/*.gql.**.js'
```

Comment:
I noticed that the coverage has decreased in this PR, and I think it might be due to `.gql.ee.js` or `.gql.js` files being imported by default during our CI which makes them "covered". You'll notice on [the failure](https://coveralls.io/builds/40065394) for this other job that it is `.gql.ce.js` files that are reported as missing coverage.

Should we _only_ ignore `.gql.ce.js` files then?